src/cssModulesLoader.ts

```diff
@@ -0,0 +1,27 @@
+export const classesMap = new Map<string, { [index: string]: string }>();
```

Comment:
Pending review, merge and release of https://github.com/dojo/webpack-contrib/pull/224 this can be removed and references updated.